server/validators/participant.ts

```diff
@@ -117,6 +125,9 @@ export const ParticipantSchema = yup
       .required(errorMessage)
       .matches(/^[A-Z]\d[A-Z]\s?\d[A-Z]\d$/, 'Postal code must be in format A1A 1A1'),
 
+    indigenous: yup.string().oneOf(yesOrNo, 'Indigenous should be Yes or No'),
```

Comment:
the docs say this question isn't required, should confirm

Reply:
It is optional. It could be debatable to treat 'Prefer not to answer' to be equal to 'unknown'.

Reply:
changed value to `preferNotToAnswer`

Reply:
Same with this one. We've determined that this question is optional so the options are
Yes
No
Prefer not to answer (Backend -> Prefer not to answer)
Blank (Backend -> Unknown)

Reply:
Then, optional yes or no can have four options, 'Yes, No, Prefer not to answer, Unknown' whereas required one has just 'Yes' or 'No'.

Capitalization for the value is a little messy. Some fields use capital and some don't. Even there is a migration to change `interested` value to lowercase.